### PR TITLE
added simple content negotiation docs

### PR DESCRIPTION
**Pull request by TomDemeranville**

### CODE REVIEW

**Comment by TomDemeranville** [comment]
[Document content negotiation on https://github.com/ORCID/ORCID-Source](https://trello.com/c/0ZiE7EQw/4757-document-content-negotiation-on-https-githubcom-orcid-orcid-source)

File: CONTENT_NEGOTIATION.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Data is returned using the most recent stable version of the API, which is version 2.1 at the time of writing.  The data returned is the equivalent of calling https://pub.orcid.org/v2.1/{orcid}/record with the corresponding content type.  See the [v2.1 documentation](https://github.com/ORCID/ORCID-Source/blob/master/orcid-model/src/main/resources/record_2.1/README.md) for a description of the format.
+
+ORCID support for content negotiation is similar to that of DOIs - [Documentation here](https://citation.crosscite.org/docs.html). 
+
+# What is content negotiation?
+
+"Content negotiation refers to mechanisms defined as a part of HTTP that make it possible to serve different versions of a document (or more generally, representations of a resource) at the same URI, so that user agents can specify which version fits their capabilities the best. " - [Wikipedia](https://en.wikipedia.org/wiki/Content_negotiation)
+
+It is defined in [RFC2616, section 12](https://www.w3.org/Protocols/rfc2616/rfc2616-sec12.html).
```
Comment (stain) [inline CONTENT_NEGOTIATION.md]: RFC2616 is obsoleted by updated specs, use instead:

```markdown
It is defined by
[RFC7231 section 5.3.2](https://tools.ietf.org/html/rfc7231#section-5.3.2) and 
[RFC7231 section 3.4](https://tools.ietf.org/html/rfc7231#section-3.4)
```

File: CONTENT_NEGOTIATION.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+| Type  | Accept Header | 
+| ------------- | ------------- | 
+| ORCID XML  | application/xml  |
+| ORCID XML  | application/orcid+xml  |
+| ORCID XML  | application/vnd.orcid+xml  |
+| ORCID JSON  | application/json  |
+| ORCID JSON  | application/json  |
+| ORCID JSON  | application/vnd.orcid+json  |
+| Schema.org JSON-LD | application/ld+json |
```
Comment (stain) [inline CONTENT_NEGOTIATION.md]: Also add:

```markdown
| [RDF Turtle](https://www.w3.org/TR/turtle/) | text/turtle |
| [RDF N-Triples](https://www.w3.org/TR/n-triples/) | application/n-triples |
| [RDF/XML](https://www.w3.org/TR/rdf-syntax-grammar/) | application/rdf+xml |
```

**Comment by TomDemeranville** [comment]
Updated.  Thanks for the comments!

Tom Demeranville
Technology Advocate
ORCID Inc
https://orcid.org/0000-0003-0902-4386

**Pull request merged**